main/src/io/github/iltotore/iron/RefinedTypeOps.scala

```diff
@@ -76,8 +78,8 @@ trait RefinedTypeOpsImpl[A, C, T]:
    * @return a [[Right]] containing this value as [[T]] or a [[Left]] containing the constraint message.
    * @see [[fromIronType]], [[option]], [[applyUnsafe]].
    */
-  inline def either(value: A)(using constraint: Constraint[A, C]): Either[String, T] =
-    Either.cond(constraint.test(value), value.asInstanceOf[T], constraint.message)
+  inline def either(value: A): Either[String, T] =
+    Either.cond(rtc.test(value), value.asInstanceOf[T], rtc.message)
```

Comment:
Same as my comment above. Should we drop the inlining for these methods in here as well?

Reply:
I don't see any reason to keep it.